### DISCUSSION TITLE
Welcome to backstage-plugins Discussions!

### DISCUSSION THREAD

**Discussion by jvilimek** [Oriflame/backstage-plugins#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by padupe** [comment]
Hi!
I've been working with Backstage for over 1 year. I helped with the initial implementation at @madeiramadeirabr and I am currently on the team responsible for implementing Backstage at Raízen.

I was interested in what the plugin could offer the community and I hope to contribute to its improvement every day.
